smurfs/src/components/App.css

```diff
@@ -1,28 +1,320 @@
 .App {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.smurf-family {
+  position: absolute;
+  z-index: 400;
+  width: 800px;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+  top: 76%;
+  position: fixed;
+  user-select: none;
+  pointer-events: none;
+}
+
+.background2 {
+  width: 100%;
+  height: 50%;
+  background: rgb(223, 191, 52);
+  position: absolute;
+  left: 0;
+  top: 49%;
+  z-index: 200;
+  position: fixed;
+}
+
+.background {
+  width: 100%;
+  height: 50%;
+  background: rgb(233, 222, 67);
+  background: -moz-radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  background: -webkit-radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  background: radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#e9de43",endColorstr="#ffd100",GradientType=1);
+  position: absolute;
+  left: 0;
+  top: 50%;
+  z-index: 200;
+  position: fixed;
 }
 
 .App-header {
-  background-color: #222;
-  height: 150px;
+  text-align: center;
+  width: 99%;
+  margin-top: 50px;
+  position: absolute;
+  z-index: 400;
+  user-select: none;
+  pointer-events: none;
+}
+
+.smurfs-logo {
+  width: 350px;
+}
+
+.title {
+  color: white;
+  font-family: 'Luckiest Guy', cursive;
+  letter-spacing: 5px;
+  font-size: 4rem;
+}
+
+.smurf-container {
+  margin: 150px auto;
+  background-color: white;
+  box-shadow: 0px 10px 20px rgba(22, 22, 22, 0.144);
+  border-radius: 30px;
+  padding: 50px;
+  width: 500px;
+  height: auto;
+  position: relative;
+  z-index: 300;
+  transition: 0.5s ease-in-out;
+}
+
+.smurf-title {
+  font-size: 1.1rem;
+  color: rgb(46, 139, 245);
+  text-transform: uppercase;
+  font-weight: 700;
+  padding-bottom: 10px;
+}
+
+.grumpy {
+  position: absolute;
+  position: fixed;
```

Comment:
could cleanup by removing the position: absolute, assuming you wanted fixed (as it was last and overrides the absolute anyway) but I mean definitely not the end of the world at all, but in terms of css hygiene, that would be something to tidy up down the road.